tests/sentry/auth/test_access.py

```diff
@@ -225,8 +229,22 @@ def test_has_access(self):
         assert result.has_team_access(self.team)
         assert result.teams == [self.team]
 
-    def test_no_access(self):
+    def test_no_access_due_to_no_installation_unowned(self):
         result = access.from_sentry_app(self.proxy_user, self.out_of_scope_org)
+        assert not result.has_team_access(self.team)
+        assert not result.has_team_access(self.team2)
+        assert not result.has_team_access(self.out_of_scope_team)
+
+    def test_no_access_due_to_no_installation_owned(self):
+        result = access.from_sentry_app(self.out_of_scope_proxy_user, self.out_of_scope_org)
+        assert not result.has_team_access(self.team)
+        assert not result.has_team_access(self.team2)
+        assert not result.has_team_access(self.out_of_scope_team)
+
+    def test_no_access_due_to_invalid_user(self):
```

Comment:
@dcramer I can confirm this test fails without Armin's fix: https://github.com/getsentry/sentry/pull/19950

Reply:
realized i typo'd "confirmed" as "confirm" :)